dcf/tests/test_commands.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding:utf-8 -*-
+
+from django.contrib.auth import get_user_model
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+from django.core import management
+from django.test import TestCase
+from django.core import mail
+
+from dcf.models import Item, Group, Section
+from StringIO import StringIO
+
+
+class GenericViewTestCase(TestCase):
+
+    def setUp(self):
+
+        management.call_command('loaddata', 'dcf/tests/init_data_for_command_test.json', verbosity=0)
+
+
+class TestCommands(GenericViewTestCase):
+
+    def test_alert(self):
+
+        mail.outbox=[]
+
+        out = StringIO()
```

Comment:
I think we need to check:
1) Outdated item link are in mail body
2) Non outdated item link not in mail body